src/App.js

```diff
@@ -1,8 +1,25 @@
-import React from 'react'
+import React, { Component } from 'react'
+import { BrowserRouter, Route, Switch } from 'react-router-dom'
 
-import { PostList } from 'components/PostList/PostList'
+import Home from './views/Home/Home'
+import About from './views/About/About'
 
-//eslint-disable-next-line
-import styles from './App.module.scss'
+import Header from './components/Header/Header'
 
-export const App = () => <PostList />
+import './css/app.general.scss'
+
+class App extends Component {
+  render() {
+    return (
+      <BrowserRouter>
+        <Header />
```

Comment:
We can have a different `Header` on different pages. Please move `Header inside of page`